--- lib/Makefile~	Thu Oct  4 00:09:43 2001
+++ lib/Makefile	Thu Oct  4 00:15:49 2001
@@ -125,7 +125,7 @@
 INSTS =
 
 SRCS = add_site.c addrmchusr.c aliasdb.c atonum.c basename.c can_access.c \
-	can_open.c chloc.c date_util.c elm_access.c errno.c expand.c \
+	can_open.c chloc.c date_util.c elm_access.c elmrc.c errno.c expand.c \
 	fast_getpw.c figadrssee.c gcos_name.c get_tz.c getarpdate.c \
 	getfullnam.c gethostname.c getword.c header_cmp.c in_list.c \
 	initcommon.c ldstate.c len_next.c $(MCSRC) mail_gets.c mailfile.c \
@@ -137,7 +137,7 @@
 	validname.c
 
 OBJS = add_site.o addrmchusr.o aliasdb.o atonum.o basename.o can_access.o \
-	can_open.o chloc.o date_util.o elm_access.o errno.o expand.o \
+	can_open.o chloc.o date_util.o elm_access.o elmrc.o errno.o expand.o \
 	fast_getpw.o figadrssee.o gcos_name.o get_tz.o getarpdate.o \
 	getfullnam.o gethostname.o getword.o header_cmp.o in_list.o \
 	initcommon.o ldstate.o len_next.o mail_gets.o mailfile.o \
